### PR TITLE
#143 MWE metric has impediments

### PR DESCRIPTION
**Pull request by uurl**
**Issue**

MWE metric has impediments. Need to wait for resolving these impediments, add MWE to reports list and cover it by tests.

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#243](https://github.com/yegor256/jpeek/issues/243) is now in scope, role is `REV`

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` is banned at [#243](https://github.com/yegor256/jpeek/issues/243); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by uurl** [comment]
@rultor merge

File: src/main/resources/org/jpeek/metrics/MWE.xsl [hunk 1]
```diff
@@ -85,7 +85,8 @@ SOFTWARE.
       </xsl:for-each>
     </xsl:variable>
     <xsl:copy>
-      <xsl:attribute name="value" select="max($OiDi/oidi/@v)"/>
+      <!--xsl:attribute name="value" select="max($OiDi/oidi/@v)"/-->
```
Comment (yegor256) [inline src/main/resources/org/jpeek/metrics/MWE.xsl]: @uurl what is the point of this comment? Either delete it or explain what should be done with it.

**Comment by 0crat** [comment]
The job [#243](https://github.com/yegor256/jpeek/issues/243) is now out of scope

**Pull request closed without merging**